partner-chains-cli/src/config.rs

```diff
@@ -265,6 +266,12 @@ impl CardanoNetwork {
 	pub fn to_id(&self) -> u32 {
 		self.0
 	}
+	pub fn to_network_param(&self) -> String {
```

Comment:
minor naming: netowrk_param here vs network_arg in the parser

Reply:
renamed the parser, it doesn't provide the arg, just the cardano network value, anyway